Gruntfile.js

```diff
@@ -146,7 +146,9 @@ module.exports = function(grunt) {
         files: {
           // Destination: [source files]
           'out/app_engine/js/apprtc.debug.js': [
-	    'src/web_app/js/analytics.js',
+            'node_modules/jssha/src/sha.js',
```

Comment:
So instead of adding it to the compiled output you want them to be served on GAE behind a handler and let AppRTC make a HTTP request for it? What do we gain from that in this case? (I do understand the benefit of callstats.min.js as it's hosted elsewhere and is not available via NPM).